metabase/metabase
# feat(sdk): Use stable IDs in SDK components

Uses https://github.com/metabase/metabase/pull/47094 to allow for stable IDs in SDK.

**_How to test_**
Now, you should just be able to use an entity ID in the `InteractiveQuestion` or `InteractiveDashboard` components and it should ✨ just work ✨.

Right now, the best way to get an entity ID is to examine the network request when you are loading a dashboard or a question. When you go to a dashboard or question page, open up the `Network` Devtools tab, find a dashboard/question API call that loads the data, and then find the `entity_id` entry. You can then use that key for testing.

Ideally we'll have a _much_ easier way of getting this key in the future. For now, this is what we have to do 🙃 

<img width="872" alt="image" src="https://github.com/user-attachments/assets/23702025-9fb8-4e06-aeb7-c33389e667a1">

<img width="332" alt="image" src="https://github.com/user-attachments/assets/d3c5e2f1-7057-412d-bda6-42966e23de41">
